### ISSUE TITLE
Have ability to lock accordion location 

### ISSUE THREAD

**Issue by kmun** [LGSInnovations/sigplot#56]
**Is your feature request related to a problem? Please describe.**
I really like the accordion feature, but would like to be able to lock the according to the location on the X Axis and with a given width that I put it at and not let the user change it when zooming and panning.  We find the users will inadvertently select one of the colored vertical accordion bars when zooming which then changes them. 

**Describe the solution you'd like**
See above

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by maihde** [comment]
This is a good idea for an enhancement.  Accordion already supports 'prevent_drag' and adding a 'prevent_resize' would be a nice complement to it.  I'll try to get that done soon.

**Comment by maihde** [comment]
@kmun I checked and using `prevent_drag` in the accordion constructor actually locks both the width and the position (i.e. the accordion can only be modified by code and not user interaction)

```
var accordion = new sigplot_plugins.AccordionPlugin({
        prevent_drag: true
});
```

To allow more flexibility I added `prevent_resize` and `prevent_move` to allow each of the various modes to be turned off individually.

**Comment by mrecachinas** [comment]
Fixed in #59.

**Comment by kmun** [comment]
Thanks.
